### ISSUE TITLE
Handle when JS is not available on the browser

### ISSUE THREAD

**Issue by jonasmalacofilho** [ITDP/the-online-brt-planning-guide#53]
The availability of JS is a requirement, mostly because without it we couldn't render equations efficiently.
- [x] **critical**: display a warning with `<noscript></noscript>`
- [x] display a warning or load message until the navigation script has been run
- [x] _investigate_ if and how could we render math without JS
- [x] _investigate_ if and how could we handle the navigation menu/table of contents

**Comment by jonasmalacofilho** [comment]
@craice, can you "draw" the warning element?

**Comment by lehonma** [comment]
Can we "translate" TeX equations to HTML text? (They won't be pretty...but at least they'll be readable)
Else I don't see a solution without generating some *.PNGs

**Comment by jonasmalacofilho** [comment]
We kind of can...  Or even generate SVGs or PNGs. 

We should wait for the math-heavy chapters to be merged though, I'm worried about if  `mathjax-node` will be able to handle the load.

**Comment by lehonma** [comment]
I found a solution when the script fails to load...we could use the following solution:

```
<script src="foo.js" onload="onload();"/>
<script>
function onload()
{
//Remove "loading" element
}
</script>
```

**Comment by jonasmalacofilho** [comment]
That's very interesting!

It makes more sense to inline this behavior in the case of the nav script, but a general approach like this might be very useful for other critical scripts.

**Comment by craice** [comment]
```
...
<header>
  <ul>...</ul>
  <div style="background-color: #F33; color: #FFF; text-align: center; padding: 4px; font-family: sans-serif; font-size: 12px;">
    JavaScript is not enabled!
  </div>
</header>
...
```

**Comment by lehonma** [comment]
Error on load nav implemented on #f573377
